src/EditorFeatures/CSharpTest2/Recommendations/AndKeywordRecommenderTests.cs

```diff
@@ -553,5 +551,25 @@ public enum Goo
     Second
 }");
         }
+
+        [WorkItem(51431, "https://github.com/dotnet/roslyn/issues/51431")]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestAfterParenthesizedPattern1()
+        {
+            await VerifyKeywordAsync(
+@"
+expr is (not []) $$");
+        }
+
+        [WorkItem(51431, "https://github.com/dotnet/roslyn/issues/51431")]
```

Comment:
@CyrusNajmabadi Is this `WorkItem` right? I am not sure. The same in the test above